pages/form-builder/support.tsx

```diff
@@ -0,0 +1,286 @@
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
```

Comment:
This file is very similar so see comments above

Reply:
True, almost a copy+paste. I went with this because of the routing problem above.
One solution I was think about is to use a next.config with rewrites (https://nextjs.org/docs/api-reference/next.config.js/rewrites) but that seemed like overkill for one case of code duplication